app/models/BasicPokemonDetail.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-let mongoose = require('mongoose');
+var mongoose = require('mongoose');
 
-let BasicPokemonDetailSchema = new mongoose.Schema({
+var BasicPokemonDetailSchema = new mongoose.Schema({
     pokemonID: {
         type: Number,
         unique: true,
```

Comment:
I just realized that there is even a bunch of outdated files to be deleted. Maybe it is better to create a separate branch for cleaning things.